portfolio/src/main/webapp/index.html

```diff
@@ -61,14 +60,38 @@ <h1 class="section-header about-me"><span>About Me</span></h1>
             </p>
             <div id="skills">
               <div class="row">
-                <span id="Java" class="dot" onmouseover="toggleDot('Java', 5)" onmouseout="toggleDot('Java', 5)">Java</span>
-                <span id="Python" class="dot" onmouseover="toggleDot('Python', 3)" onmouseout="toggleDot('Python', 5)">Python</span>
-                <span id="Cpp" class="dot" onmouseover="toggleDot('Cpp', 2)" onmouseout="toggleDot('Cpp', 5)">Cpp</span>
+                <span id="Java" class="skill-dot" 
+                  onmouseover="toggleSkillDot(this, {years: 5})" 
```

Comment:
Sorry to bring this up again, but I guess the issue I have with using the word "dot" is that it doesn't give me any semantic information. If we were just talking about your website and I didn't have a picture of it, if you said "skill dot", I would have no idea what it is supposed to be.

Reply:
Good point. Each of these items are just programming languages displayed on circles. Would "programming-language" be descriptive enough instead?